orca-api/src/main/java/com/netflix/spinnaker/orca/api/pipeline/models/ExecutionStatus.java

```diff
@@ -71,6 +71,10 @@ public final boolean isComplete() {
     return complete;
   }
 
+  public final boolean isSkipped() {
+    return this.toString() == "SKIPPED";
```

Comment:
I believe you can use something like `SKIPPED.equals(this)`

Reply:
Excellent, checking the previous code I was looking by something similar